instrumentation/google.golang.org/grpc/otelgrpc/test/interceptor_test.go

```diff
@@ -401,6 +431,74 @@ func TestStreamClientInterceptorOnBIDIStream(t *testing.T) {
 	_ = streamClient.CloseSend()
 }
 
+func TestStreamClientInterceptorEvents(t *testing.T) {
+	testCases := []struct {
+		Name     string
+		NumMsgs  int
+		NoEvents bool
+	}{
+		{Name: "With events", NumMsgs: 1, NoEvents: false},
+		{Name: "No events", NumMsgs: 1, NoEvents: true},
+		{Name: "10 messages with events", NumMsgs: 10, NoEvents: false},
+		{Name: "10 messages no events", NumMsgs: 10, NoEvents: true},
```

Comment:
I think we should also have tests for only sent events, and only received events.
Also, why the `10 messages` tests? There's no real reason to have a different behavior between 1 and 10 messages.

Reply:
Updated.